app/invocation/invocation.css

```diff
@@ -50,6 +50,11 @@
   height: 18px;
 }
 
+.invocation-action-card .terminal-container {
+  padding: 8px;
```

Comment:
8px seems a little tight on the sides.

maybe something more like 
```
padding: 8px 32px;
```

Reply:
sg, done -- the additional padding uncovered an issue that the top bar of the terminal was "ignoring" the padding since it is absolutely positioned -- fixed by adding a wrapper div inside `.terminal-container` and positioning relative to that div instead. Here's what it looks like now:

![image](https://user-images.githubusercontent.com/2414826/147972090-92c43404-19bd-4ba4-abe7-d566411eacc9.png)